src/CodeGen_ARM.cpp

```diff
@@ -1316,6 +1433,11 @@ string CodeGen_ARM::mattrs() const {
             separator = ",";
         }
 
+        if (target.has_feature(Target::ARMFp16)) {
+            arch_flags += separator + "+fullfp16";
```

Comment:
What minimum version of LLVM is required to support this flag?

Reply:
I was not able to identify the exact version. But this flag is supported at least LLVM10 which Halide has already dropped support.